packages/odyssey-react-mui/src/Button.tsx

```diff
@@ -133,6 +137,7 @@ const Button = ({
   translate,
   type = "button",
   variant,
+  ...nativeButtonProps
```

Comment:
I'd prefer we don't use `...restProps` anywhere in our codebase. It will make it difficult for us to know what props people are passing and can easily lead to a breaking of accessibilty.

Can we be explicit about which props we want to allow instead? Props like `ref` and `className` are ones we absolutely don't want to expose.